code-reviews/recipes/Terraform.md

```diff
@@ -1,18 +1,23 @@
-# Terraform Code Reviews
+## Code Reviews
+### Style Guide
+- Adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html)
+- Ensure all used providers are versioned to prevent breaking changes in future. [docs on versioning providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
+- main.tf: This file usually contains provider configuration, backend configuration, imports to the modules to use and eventually some common resources that was not isolated into a specific file. It is absolutely possible to have the whole infrastructure defined in this file itself, suggest to split it into other *.tf files
+- README.md - Always useful to document what the Terraform project does
 
-## Style guide
+### Linters
+1. [tflint](https://github.com/terraform-linters/tflint) - As of now tflint doesn't have Azure specific rules, it can be used to inspect  general Terraform specific issues.  
 
-[CSE](../../CSE.md) developers adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html).
+### Useful extensions for VS Code and Visual Studio
```

Comment:
Useful extensions for VS Code and Visual Studio -> VS Code and Visual Studio Extension

to follow the same format as the other recipes

Reply:
@TessFerrandez - Changed it to VS Code Extensions.

Reply:
VS Code Extensions is perfect... my comment was solely based on that the previous title mentioned Visual Studio